Python/openai/README.md

```diff
@@ -8,20 +8,24 @@ Example Python and OpenAI application using [Autoblocks](https://www.autoblocks.
 
 ## Setup
 
-* Install [`pyenv`](https://github.com/pyenv/pyenv)
```

Comment:
getting rid of this b/c people handle virtualenvs differently and poetry will just create one for you if you're not already in one